apps/pie-storybook/stories/contribution/overview.mdx

```diff
@@ -0,0 +1,220 @@
+```
+
+### Set the component status
+
+In `package.json`:
+
+```json
+"pieMetadata": {
+    "componentStatus": "alpha"
+}
+```
+
+Please do not change the status without agreement from the Design System team. All new components start in `alpha`. See [here](https://pie.design/components/component-status/#status-descriptions) for more information.
+
+## 6. Component package anatomy
+
+This is the typical structure for a PIE component:
+
+```
+component-name/
+├── dist/
+├── src/
+│   ├── component-name.scss
+│   ├── defs.ts
+│   ├── defs-react.ts
+│   ├── index.ts
+│   └── react.ts
+│
+├── test/
+│   ├── accessibility/
+│   │   └── component-name.spec.ts
+│   ├── component/
+│   │   └── component-name.spec.ts
+│   ├── helpers/
+│   │   └── page-object/
+│   │       └── selectors.ts
+│   └── visual/
+│       └── component-name.spec.ts
+│
+├── custom-elements-manifest.config.mjs
+├── package.json
+├── README.md
+├── tsconfig.json
+├── turbo.json
+├── vite.config.js
+├── playwright-lit-visual.config.ts
+└── playwright-lit.config.ts
+```
+
+### File breakdown
+
+- **package.json** - Metadata, dependencies, `pieMetadata`
+- **README.md** - Usage and development notes. This is imported into Storybook as the `Overview` page for a component.
+- **tsconfig.json** - TypeScript configuration
+- **vite.config.js** - Vite dev/build config
+- **turbo.json** - Turborepo monorepo config
+- **custom-elements-manifest.config.mjs** - Custom Elements manifest setup. [More information](https://github.com/webcomponents/custom-elements-manifest).
+- **playwright-lit-visual.config.ts** - Visual testing Playwright config
+- **playwright-lit.config.ts** - General Playwright test config
+
+### Source directory (`src/`)
+
+- `component-name.scss` - Component styles
+- `defs.ts` - Type definitions
+- `defs-react.ts` - React types used by the React wrapper
+- `index.ts` - Component entry and exports
+- `react.ts` - Component entry and exports for React (this is an untracked file creating during build)
+
+### Test directory (`test/`)
+
+- `accessibility/component-name.spec.ts` - Accessibility checks
+- `component/component-name.spec.ts` - Component browser tests
+- `helpers/page-object/selectors.ts` - Page object helpers for tests
+- `visual/component-name.spec.ts` - Visual regression tests
+
+## 7. Testing standards
+
+Please read our [testing guide](?path=/docs/contribution-testing-overview--docs).
+
+## 8. Create a pull request
+
+### Branch naming
+
+Use format: `dsw-123-my-feature` where `dsw-123` is the ticket ID.
+
+### Commit messages
+
+Use `yarn cz` to generate commits.
+
+Format:
+
+```
+type(scope): DSW-123 your message
+```
+
+### Pull request title
+
+Format:
+
+```
+type(scope): DSW-123 your PR title
+```
+
+Complete the [pull request template](https://github.com/justeattakeaway/pie/blob/main/.github/pull_request_template.md). Otherwise DangerJS checks will fail.
+
+---
+
+## 9. Add a changeset (if needed)
```

Comment:
I would put this step before making a PR (as it'll need to be commited) – so at step 8